plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/DBNModel.java

```diff
@@ -271,6 +271,9 @@ private static NodePath getNodePath(@NotNull String path) {
                 break;
             }
         }
+        if (path.contains(DBNModel.PATTERN_ESCAPE_TOKEN)) {
```

Comment:
It's used here and assumes _something_ about how paths are escaped. Check this within the `decodeNodePath` method and don't introduce new constants - it introduces unnecessary noise.